docker/Dockerfile

```diff
@@ -189,10 +271,10 @@ RUN pip install --no-deps --no-cache-dir /source/dist/*.whl && \
 FROM base as final
 ARG PYTHON_VER
 
-# Copy from "dependencies" the required python libraries and binaries
-COPY --from=dependencies /usr/local/lib/python${PYTHON_VER}/site-packages /usr/local/lib/python${PYTHON_VER}/site-packages
-COPY --from=dependencies /usr/local/bin /usr/local/bin
-COPY --from=dependencies /source/dist/*.whl /tmp
+# Copy from "build-nautobot" the required python libraries and binaries
+COPY --from=build-nautobot /usr/local/lib/python${PYTHON_VER}/site-packages /usr/local/lib/python${PYTHON_VER}/site-packages
+COPY --from=build-nautobot /usr/local/bin /usr/local/bin
```

Comment:
Could still copy these two from `dependencies`, I think - only the wheel below actually needs to come from `build-nautobot`?